src/pages/my-courses/MyCourses.constants.tsx

```diff
@@ -1,7 +1,17 @@
+import { CourseFilters } from '~/types'
+
 export const defaultResponse = { items: [], count: 0 }
 
 export const courseItemsLoadLimit = {
   tablet: 6,
   mobile: 6,
   default: 6
 }
+
+export const defaultFilters: CourseFilters = {
```

Comment:
I suggest you can use existing `coursesDefaultFilters` and modifies them instead creation another one obj

Reply:
Modified `coursesDefaultFilters` and deleted `defaultFilters` from `MyCourses.constants`.
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/104922087/7c8f3853-e4b0-4b53-b6b0-de7f9c0824a0)